pages/express-relay/how-express-relay-works.mdx

```diff
@@ -1 +1,44 @@
-# Design Overview
\ No newline at end of file
+# How Express Relay Works
+
+Express Relay allows protocols to eliminate miner-extractable value (MEV).
+Many protocols generate MEV on a regular basis.
+For example, borrow-lending protocols provide bonuses to searchers for liquidating undercollateralized loans.
+Searchers compete for these bonuses by tipping the chain's miners or validators.
+The validators capture most of the value of the liquidation bonus via these tips, so the liquidation bonus is in essence a transfer of wealth from the protocol's users to the validators.
+
+Express Relay solves the problem of MEV by providing protocol developers with an auction primitive that they can use to prioritize access to permissionless operations.
+Developers specify a set of operations in their protocol that must be accessed through Express Relay.
+Searchers then participate in an off-chain auction to access these operations.
+Their bids in the auction determine the priority of their transactions, i.e., the order in which their transactions will be executed.
+The winners transactions are forwarded to the blockchain, which both pays their bid and executes the operation.
+The profits of the auction are then split between the integrated protocol and other participants in Express Relay.
+
+![](images/express_relay/express_relay_schematic.svg)
+
+The diagram above shows how Express Relay changes the MEV landscape for a liquidation.
+In the status quo (left), Searchers tip miners in order to guarantee that their liquidation transaction lands on-chain.
+Their transaction directly interacts with the protocol exposing the liquidation opportunity, and the liquidation bonus flows back to the Searcher.
+With Express Relay (right), Searchers submit bids for their transaction to the Express Relay auction.
+The auction submits the winning bids to the blockchain, where the transactions are processed by the Express Relay Entrypoint before being forwarded on to the integrated protocol.
+The Express Relay Entrypoint collects payment from the Searchers and forwards a share of the revenue back to the integrated protocol.
+
+FIXME: I think the diagram is wrong (specifically the liquidation bonus going back to express relay)
+
+## Which protocols can use Express Relay?
+
+Any protocol with permissionless and valuable operations can use Express Relay.
+These operations generate MEV, as the validators control which searchers can access them.
```

Comment:
These two sentences can be rewritten as "Any protocol exposing valuable operations that generate MEV "opportunities" can use Express Relay